src/keycloak/chart/templates/keycloak-logging-alerting-rules.yaml

```
@@ -0,0 +1,51 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+{{- if .Values.detailedObservability }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "keycloak.fullname" . }}-logging-alerts
+  namespace: {{ .Release.Namespace }}
+  labels:
+    prometheus: kube-prometheus-stack-prometheus
+    app.kubernetes.io/name: {{ include "keycloak.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: keycloak
+spec:
+  groups:
+    - name: keycloak-logging-alerts
+      rules:
+        - alert: KeycloakRealmModificationsDetected
+          expr: uds_keycloak:realm_modifications > 0
+          for: 0m
+          labels:
+            severity: warning
+            service: keycloak
+            source: loki
+          annotations:
+            summary: "Keycloak realm modifications detected"
+            description: "Realm modifications detected in Keycloak logs for realm {{`{{ $labels.realmId }}`}}. Value={{`{{ $value }}`}} (non-zero means changes observed recently)."
```

```suggestion
            description: "Realm {{`{{ $labels.realmId }}`}} had {{`{{ $value }}`}} admin events in the last 5 minutes."
```
More accurate description of this new alert